hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/OzoneClientConfig.java

```diff
@@ -486,6 +540,22 @@ public void setDatastreamPipelineMode(boolean datastreamPipelineMode) {
     this.datastreamPipelineMode = datastreamPipelineMode;
   }
 
+  public void setHBaseEnhancementsEnabled(boolean isHBaseEnhancementsEnabled) {
+    this.hbaseEnhancementsEnabled = isHBaseEnhancementsEnabled;
+  }
+
+  public boolean getHBaseEnhancementsEnabled() {
+    return this.hbaseEnhancementsEnabled;
+  }
+
+  public void setFsHsyncEnabled(boolean fsHsyncEnabled) {
+    this.fsHsyncEnabled = fsHsyncEnabled;
+  }
+
+  public boolean getFsHsyncEnabled() {
```

Comment:
not used anywhere in the production code.
It seems to me ozone.client.fs.hsync.enabled is not needed.

Reply:
It is supposed to be used by the client instead of `ozone.fs.hsync.enabled` to enforce the check in one place. Need adjustment.